UnityMcpBridge/UnityMcpServer~/src/tools/manage_scene.py

```
@@ -66,4 +58,4 @@ def _coerce_int(value, default=None):
             return response if isinstance(response, dict) else {"success": False, "message": str(response)}
 
         except Exception as e:
-            return {"success": False, "message": f"Python error managing scene: {str(e)}"}
\ No newline at end of file
+            return {"success": False, "message": f"Python error managing scene: {str(e)}"}
```

Ruff (0.13.1)

Use explicit conversion flag

Replace with conversion flag

(RUF010)